spec/03-plugins/17-ip-restriction/02-access_spec.lua

```diff
@@ -248,7 +248,7 @@ for _, strategy in helpers.each_strategy() do
         })
         local body = assert.res_status(403, res)
         local json = cjson.decode(body)
-        assert.same({ message = "Your IP address is not allowed" }, json)
+        assert.is_true(string.find(json, "IP address not allowed"))
```

Comment:
We could use `assert.matches()` with the special pattern.

Reply:
Can you send a link to the docs for assert.matches()?  I can't find it.